actionpack/test/controller/support/custom_assertions.rb

```diff
@@ -0,0 +1,3 @@
+def assert_filtered_out(params, key)
+  assert !params.has_key?(key), "key #{key.inspect} has not been filtered out"
+end
```

Comment:
@carlosantoniodasilva makes sense.  As this assertion is specific to controller and specifically parameters can we call it something like.

```
module ActionContoller
  module Assertions
    module Parameters
      def assert_filtered_out(params, key)
        assert !params.has_key?(key), "key #{key.inspect} has not been filtered out"
      end
    end
  end
end

```
Then we need to include

`include ActionController::Assertions::Parameters`

and the file can be named as

`support/params_assertions.rb` wdyt?